src/_P098_PWM_motor.ino

```diff
@@ -29,6 +29,8 @@
 # define P098_LIMIT_SWB_DEBOUNCE PCONFIG(5)
 # define P098_ENC_TIMEOUT        PCONFIG(6)
 # define P098_PWM_DUTY           PCONFIG(7)
+# define P098_VIRTUAL_SPEED      PCONFIG(8)
+# define P098_POS_0_SUPPLEMENT   PCONFIG(9)
```

Comment:
Ah found a bug....
PCONFIG(8) and PCONFIG(9) are invalid.
These will overwrite other settings.
You can use PCONFIG_LONG() upto 3 (4 elements)